integration/maptest/map.go

```diff
@@ -55,6 +55,7 @@ var AllTests = TestTable{
 	{"LeafHistory", RunLeafHistory},
 	{"Inclusion", RunInclusion},
 	{"InclusionBatch", RunInclusionBatch},
+	{"WriteStress", RunWriteStress},
```

Comment:
This table is what the default RunIntegration test iterates through. 

It looks like you've added a new entry point in `map_test.go` We should probably pick just one of them.  Otherwise, this test will be run twice.

Reply:
That's by design, and it won't be run twice since the new entry point will skip by default.
I added a new target  so that I can run stress tests *only* and be able to tune the level.

Reply:
Not sure if I completely follow. 
if we pass `--stress` and run `go test ./integration` the following tests will be run:

```
TestIntegration/etc...
TestIntegration/InclusionBatch
TestIntegration/WriteStress
TestMapWriteStress
```

If you'd like to only run one test does 
`go test ./integration -run TestIntegration/WriteStress` accomplish the job?

Reply:
That's probably good enough.